pnet_packet/benches/perf.rs

```diff
@@ -0,0 +1,73 @@
+use std::{fs::File, os::raw::c_int, path::Path};
```

Comment:
The code in this file seems... Pretty unrelated to libpnet? It seems like all of this should be in some other crate?

Reply:
it was a flamegraph that allowed me to see where pnet was locking up